### PR TITLE
 add new endpoint for package of spog ui

### PR DESCRIPTION
**Pull request by bxf12315**
issue https://github.com/trustification/trustification/issues/829

### CODE REVIEW

File: bombastic/index/src/sbom/mod.rs [hunk 1]
```diff
@@ -678,8 +678,8 @@ mod tests {
 
     const TESTDATA: &[&str] = &[
         "../testdata/ubi9-sbom.json",
-        "../testdata/kmm-1.json",
-        "../testdata/my-sbom.json",
+        // "../testdata/kmm-1.json",
+        // "../testdata/my-sbom.json",
```
Comment (lulf) [inline bombastic/index/src/sbom/mod.rs:682]: Should probably be reenabled

**Review by lulf — COMMENTED** [review]
LGTM, just one comment on a few lines that probably wasn't meant to be commented out.

File: bombastic/index/src/sbom/mod.rs (hunk 1 above)
Comment (bxf12315) [reply bombastic/index/src/sbom/mod.rs]: yes, I will enable it.

**Review by ctron — APPROVED** [review]
Looks really good. Great work! Thank you!

**Review by lulf — APPROVED** [review]

**Pull request merged**